### PR TITLE
Add links to README

### PR DESCRIPTION
**Pull request by chrisderwahre**
There are no links to the services mentioned in the README file, so its harder for new people to find these places.

I added a link that never expires to the discord in this PR so new member can easy join just by clicking on the text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,21 @@ You can [create an issue to contact XboxDev maintainers](https://github.com/Xbox
 
 We currently provide the following services:
 
-* Discord for communication
-* XboxDevWiki to document research
-* GitHub organization to host tools and files
+* [Discord](https://discord.gg/XUDsBYD) for communication
+* [XboxDevWiki](http://xboxdevwiki.net/Main_Page) to document research
```
Comment (JayFoxRox) [inline README.md]: Please remove `Main_Page`

**Review by dracc — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -14,21 +14,21 @@ You can [create an issue to contact XboxDev maintainers](https://github.com/Xbox
...
 
-* XboxDev on Discord
+* [XboxDev on Discord](https://discord.gg/XUDsBYD)
 
 and for closely related projects:
 
-* #xqemu on Freenode IRC (Webchat)
-* Cxbx-Reloaded on gitter.im
+* [#xqemu on Freenode IRC](irc://chat.freenode.net/xqemu) ([Webchat](https://webchat.freenode.net/?channels=%23xqemu))
+* [Cxbx-Reloaded on gitter.im](https://gitter.im/Cxbx-Reloaded/Lobby)
```
Comment (mborgerson) [inline README.md]: Probably the "Original Xbox Emulation" Discord is a better way to reach the CXBX-R guys. @PatrickvL and @LukeUsher can confirm

Comment (JayFoxRox) [reply README.md]: The contact information was copied from XboxDevWiki and is from early 2017.
The Cxbx-Reloaded README does indeed list both now (only Discord back then).

From experience I can say that the gitter is dead. I also recently asked and was told they primarily use Discord.
I wonder if we should refomat the entire paragraph slightly to something like:

> - [XQEMU](xqemu-web-or-project): [#xqemu on Freenode IRC](link) ([Webchat](link))
> - [Cxbx-Reloaded](cxbx-r-web-or-project): [Discord](link) or [gitter.im](link)

Code:

```
- [XQEMU](xqemu-web-or-project): [#xqemu on Freenode IRC](link) ([Webchat](link))
- [Cxbx-Reloaded](cxbx-r-web-or-project): [Discord](link) or [gitter.im](link)
```

Comment (LukeUsher) [reply README.md]: I'm of two minds weather to even link the Gitter at all at this point: Everyone just gravitated towards discord. I do like this suggested formatting though, linking to both the project pages (or websites) and a discussion location is a good idea.

File: README.md [hunk 3]
```diff
@@ -14,21 +14,21 @@ You can [create an issue to contact XboxDev maintainers](https://github.com/Xbox
 
 We currently provide the following services:
 
-* Discord for communication
-* XboxDevWiki to document research
-* GitHub organization to host tools and files
+* [Discord](https://discord.gg/XUDsBYD) for communication
+* [XboxDevWiki](http://xboxdevwiki.net) to document research
```
Comment (0DaveX) [inline README.md:18]: `[XboxDevWiki.net](http://xboxdevwiki.net) to document research`
Maybe that would help to spreading the url so ppl will memorize it more?

**Comment by 0DaveX** [comment]
The links are confirmed to be working.
Everything looks good and mergeable imho.

**Review by ghost — APPROVED** [review]

**Pull request merged**